docs/sources/http_api/data_source.md

```diff
@@ -194,15 +308,6 @@ Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
 }
 ```
 
-**Example Response**:
```

Comment:
Why is the example response removed?

Reply:
it's not removed, I have voted it before the basic auth request and response example [here](https://github.com/grafana/grafana/blob/016c4a1d06c1e409f19a97ca29c3468e7a2e2c03/docs/sources/http_api/data_source.md#L195).

Reply:
I don't really understand what you mean, to be honest :/ When I check the diff vs master, the example Cloudwatch response is removed in your branch.